### ISSUE TITLE
Top-X для вертикальных объёмов.

### ISSUE THREAD

**Issue by Hefren** [SBProX/SBProX-Dev#1686]
Как вам идея реализовать такое?

**Comment by weeclemans** [comment]
а где скрины с примерами или видео?

**Comment by Hefren** [comment]
> Приветствую.
> А где скрины с примерами или видео?

Так, Топ-Х стандартный индикатор, только он по горизонтали подкрашивает объём. Тоже самое только вертикальные объёмы, вроде без фото очевидно всё.

**Comment by weeclemans** [comment]
да, только наглядный пример будет всегда в приоритете

**Comment by SBProX** [comment]
Не знаю, есть ли в этом смысл - по фильтрам и так видно - пусть пользователи комментируют.

**Comment by indianaxxx** [comment]
... вертикальные цветом нормально фильтруются...

**Comment by Hefren** [comment]
> ... вертикальные цветом нормально фильтруются...

Но со временем объём торгов стабильно изменяется. Этот год яркий тому пример. Сейчас месяц от месяца какую-то конкретную цифру объёма можно считать большой, а иной месяц маленькой. Если бы подсчёт был на автомате, это было бы великолепно.
